trieste/acquisition/optimizer.py

```diff
@@ -58,7 +59,8 @@ def automatic_optimizer_selector(
         return optimize_discrete(space, target_func)
 
     elif isinstance(space, Box):
-        return optimize_continuous(space, target_func)
+        num_samples = tf.minimum(2000, 500 * tf.shape(space.lower)[-1])
```

Comment:
increase this?

Reply:
I thought it didnt seem to make a difference for COCO? 1000*dim?, min 5000?

Reply:
1000*dim?, min 5000? sounds good